README.md

```diff
@@ -150,7 +150,7 @@ you are interested in working on a PR or issue opened in the previous repository
 
 ## Rust version requirements
 
-Since version 0.21.0, The Minimum Supported Rust Version (MSRV) of `ratatui` is 1.65.0.
```

Comment:
I think *"Since version 0.21.0"* should be updated to *0.23.0* as the MSRV will be bumped in the next release.

Reply:
Ah, right. Feel free to submit a PR!

Reply:
Done at #369 :)